controller/instance_manager_controller.go

```diff
@@ -462,7 +506,7 @@ func (imc *InstanceManagerController) syncInstanceManagerPDB(im *longhorn.Instan
 	return nil
 }
 
-func (imc *InstanceManagerController) cleanUpPDBOfNonExistingIMPod() error {
+func (imc *InstanceManagerController) cleanUpPDBForNonExistingIM() error {
```

Comment:
NIT: for now this is okay, but in the future we might want to move this into a gc controller, since now each IM will evaluate all PDBs in the longhorn-system namespace. Also all of the functions use copies even though they are all readonly.

Reply:
Right. Besides, we can add labels for IM PDBs then do filtering here.

Reply:
For the gc controller as we discussed before, it's also possible for other possible orphan resources cleanup.